tests/test_code_references.py

```diff
@@ -203,114 +208,123 @@ def x(my_class):
 
     assert get_references(x) == ["val", logging.info, logging.debug]
 
+    import math
+
+    def y(op_type, val):
+        if op_type == "ceil":
+            func = math.ceil
+        else:
+            func = math.floor
+
+        return func(val)
+
+    # TODO: We should also return math.ceil as a reference. It isn't
+    # returned now because it is stored in a local variable. Maybe we
+    # should return the values of local variables as they are assigned.
```

Comment:
Nice catch. I'm still not sure why we don't use `set_tos` for `STORE_FAST` to track these references. Is there a drawback?

Reply:
The only issue I can see is that we add the reference multiple times. But given that we are going to dedup the references anyway, I think we should add to `tos` for `STORE_FAST` too. I'll make the change in this PR.